test/integration/crcsuite/crcsuite.go

```diff
@@ -54,6 +54,8 @@ func FeatureContext(s *godog.Suite) {
 		CheckHTTPResponseWithRetry)
 	s.Step(`^with up to "(\d+)" retries with wait period of "(\d*(?:ms|s|m))" command "(.*)" output (should match|matches|should not match|does not match) "(.*)"$`,
 		CheckOutputMatchWithRetry)
+	s.Step(`^checking that CRC status is (Running|Stopped)$`,
```

Comment:
nicer to read as `checking that CRC is (running|stopped)$` ?

Reply:
True. Will address other comments as well. Thanks!